bsp4kt/resources/util/Serializers.kt

```diff
@@ -0,0 +1,28 @@
+package bsp4kt.util
+
+import kotlinx.serialization.DeserializationStrategy
+import kotlinx.serialization.SerializationException
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.JsonPrimitive
+import kotlinx.serialization.json.intOrNull
+
+fun <T>untaggedUnionSerializer(stringSerializer: DeserializationStrategy<T>? = null, intSerializer: DeserializationStrategy<T>? = null): (JsonElement) -> DeserializationStrategy<T> {
```

Comment:
```suggestion
fun <T> untaggedUnionSerializer(stringSerializer: DeserializationStrategy<T>? = null, intSerializer: DeserializationStrategy<T>? = null): (JsonElement) -> DeserializationStrategy<T> {
```

chyba nie było formatowane?